### DISCUSSION TITLE
Thanks for the great control library

### DISCUSSION THREAD

**Discussion by VPKSoft** [Taiizor/ReaLTaiizor#24] [Show and tell]
Hi,
I just wanted to thank you for this control library 🙂 The main window of my music software is now using the crown controls and looks better than with the basic windows forms controls. I really don't have any design talent so this helps a lot 👍
A sample of the [amp#](https://github.com/VPKSoft/amp) software with the face-lift:
![image](https://user-images.githubusercontent.com/40712699/104471166-31f42680-55c3-11eb-83ba-7502553b8564.png)

**Comment by Taiizor** [comment]
First of all, thank you for your support of my library.
I was following the projects that referenced my library and I was following your project.
It's pretty handy, even as it is now. I hope you'll make it better in time.
